src/test/java/uk/gov/companieshouse/model/validator/EndDateValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.TestData;
+import uk.gov.companieshouse.TestUtil;
+import uk.gov.companieshouse.model.Appeal;
+
+public class EndDateValidatorTest
+{
+
+    private EndDateValidator endDateValidator;
+    private Appeal appeal = new Appeal();
+
+    @BeforeEach
+    private void setup() {
+        endDateValidator = new EndDateValidator();
+        appeal = TestUtil.createAppeal(TestUtil.buildCreatedBy(),
+            TestUtil.createReasonWithIllness());
+        appeal.setCreatedAt(LocalDateTime.now());
+    }
+
+    @Test
```

Comment:
On tests we use the @DisplayName annotation to put a more verbose explanation e.g.
@DisplayName("Should return string if end date is empty")
Add to the other tests

Reply:
Missed that while writing, will add those now